src/app/header/Header.tsx

```diff
@@ -66,6 +67,9 @@ export function Header({ onSearchChange }: HeaderProps): JSX.Element {
         <GridItem colStart={3}>
           <TermButtons />
         </GridItem>
+        <GridItem colStart={4}>
+          <ExternalLinks />
```

Comment:
If you opt for going with the props over the `useState` approach, this doesn't have to be it's own component

Reply:
We _shouldn't_ consider this approach as there isn't a need for `useState` and complicates the code unnecessarily let alone accessibility issues I imagine.